RetailCoder.VBE/Navigation/CodeMetrics/CodeMetricsViewModel.cs

```diff
@@ -0,0 +1,72 @@
+﻿using NLog;
+using Rubberduck.Parsing.VBA;
+using Rubberduck.UI;
+using Rubberduck.UI.Command;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Rubberduck.Navigation.CodeMetrics
+{
+    public class CodeMetricsViewModel : ViewModelBase
```

Comment:
The class should implement `IDisposable` and the disposing pattern. That way you can deregister events from state cleanly.